account_fiscal_position_rule/__manifest__.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
```

Comment:
Remove This line [https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-11.0#tasks-to-do-in-the-migration](https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-11.0#tasks-to-do-in-the-migration)